### ISSUE TITLE
Add issue templates

### ISSUE THREAD

**Issue by ShadowJonathan** [matrix-org/matrix-rust-sdk#405]
Currently this repo is grabbing issue templates from https://github.com/matrix-org/.github/tree/master/.github/ISSUE_TEMPLATE, maybe more specialized templates under a `.github/ISSUE_TEMPLATE` folder would help here.

**Comment by poljar** [comment]
Help in what way? The templates seem to be general enough, what's exactly wrong with them and what should be improved.

As is, this issue isn't really actionable.

**Comment by ShadowJonathan** [comment]
It would help make the issues more specific to matrix-rust-sdk, as currently the templates from matrix-org generic cannot address things like matrix-rust-sdk versions in a potential better bug template ("please put the code snippet/traceback/version involved here")

**Comment by jplatte** [comment]
I think all of this doesn't really make sense for the bug report template in matrix-sdk's context:

```markdown
**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, safari]
 - Version [e.g. 22]
```

**Comment by ShadowJonathan** [comment]
Yeah, that issue template was grabbed from the matrix-org organisation .github repo, so adding issue templates specific to this repo would help.
